chalice/chalicelib/templates/components/team_status.html

```diff
@@ -1,2 +1,41 @@
 {%- set stats = status %}
-{% include "components/5_column_status.html" %}
\ No newline at end of file
+{# include "components/5_column_status.html" #}
+{%- if team_summary.all.accounts_passed > 0 -%}
+<p>
+    There are currently {{team_summary.all.accounts_passed }} accounts passing.
+</p>
+{%- endif -%}
+{%- if team_summary.all.accounts_failed > 0 -%}
+<p>
+    There are currently a total of {{ team_summary.all.issues_found }} failed resources
```

Comment:
Could check the number of failed resources to see if you need to use the singular form of "resources". Ditto for "accounts" below.